client/settings/index.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * External dependencies
+ *
+ * @format
+ */
+
+import * as SHARED from '@woocommerce/settings';
+
+/**
+ * Internal dependencies
+ */
+import * as FALLBACKS from './fallbacks';
+
+// if `getSetting` is not set, then it was not available so let's do
```

Comment:
Comment capitalization:
```suggestion
// If `getSetting` is not set, then it was not available so let's do
```